layers/meta-resin-beaglebone/conf/samples/local.conf.sample

```diff
@@ -58,3 +58,5 @@ BB_DISKMON_DIRS = "\
     ABORT,${SSTATE_DIR},100M,1K"
 
 CONF_VERSION = "1"
+
+HOSTTOOLS += "docker iptables"
```

Comment:
Can we have it detailed in the git logs why this is necessary?

Reply:
Yes, I'll add it to the logs. The reason is that host docker is required by parts of the build and iptables is also required as a dependency of the docker daemon. Without this the build fails because these tools are not available in the recipe sysroot.